ScoutSuite/providers/aws/provider.py

```diff
@@ -79,12 +79,12 @@ def preprocessing(self, ip_ranges=None, ip_ranges_name_key=None):
         # Various data processing calls
         self._check_ec2_zone_distribution()
         self._add_security_group_name_to_ec2_grants()
-        self._add_last_snapshot_date_to_ec2_volumes()
-        self._process_cloudtrail_trails(self.services['cloudtrail'])
-        self._add_cidr_display_name(ip_ranges, ip_ranges_name_key)
-        self._merge_route53_and_route53domains()
-        self._match_instances_and_roles()
-        self._match_iam_policies_and_buckets()
+        # self._add_last_snapshot_date_to_ec2_volumes()
+        # self._process_cloudtrail_trails(self.services['cloudtrail'])
+        # self._add_cidr_display_name(ip_ranges, ip_ranges_name_key)
+        # self._merge_route53_and_route53domains()
+        # self._match_instances_and_roles()
+        # self._match_iam_policies_and_buckets()
```

Comment:
Those currently crash because I'm still missing a lot of stuff in the data structure.